src/pybind/mgr/volumes/fs/exception.py

```diff
@@ -0,0 +1,10 @@
+class VolumeException(Exception):
+    def __init__(self, error_code, error_message):
+        self.errno = error_code
+        self.error_str = error_message
```

Comment:
```suggestion
        super(self, VolumeException).__init__(error_message)
```

No need to introduce another member here.